supervisor/homeassistant.py

```diff
@@ -581,9 +580,20 @@ async def check_api_state(self) -> bool:
 
         return False
 
-    async def _block_till_run(self) -> None:
+    async def _block_till_run(self, version: str) -> None:
         """Block until Home-Assistant is booting up or startup timeout."""
+        # Skeep landingpage
```

Comment:
```suggestion
        # Skip landingpage
```

Looks like another typo :)